pkg/sdk/tasks_impl_gen.go

```diff
@@ -332,6 +351,8 @@ func (r taskDBRow) convert() *Task {
 			}
 		}
 		task.Predecessors = ids
+	} else {
+		task.Predecessors = make([]SchemaObjectIdentifier, 0)
```

Comment:
Just curious, why can't we leave this as `nil`?

Reply:
I think this could be a bigger discussion on handling certain parameters in convert and other mapping functions. For now, I would leave it as is, but also can revert because predecessors are not used in the resource.